training/tf/tfprocess.py

```diff
@@ -149,77 +169,95 @@ def init(self, batch_size, macrobatch=1, logbase='leelalogs'):
 
         planes = tf.to_float(planes)
 
-        planes = tf.reshape(planes, (batch_size, 18, 19*19))
-        probs = tf.reshape(probs, (batch_size, 19*19 + 1))
+        planes = tf.reshape(planes, (batch_size, 18, 19 * 19))
+        probs = tf.reshape(probs, (batch_size, 19 * 19 + 1))
         winner = tf.reshape(winner, (batch_size, 1))
 
         self.init_net(planes, probs, winner)
 
     def init_net(self, planes, probs, winner):
-        self.x = planes  # (tf.float32, [None, 18, 19 * 19])
-        self.y_ = probs  # (tf.float32, [None, 362])
-        self.z_ = winner # (tf.float32, [None, 1])
+        self.y_ = probs   # (tf.float32, [None, 362])
+        self.sx = tf.split(planes, self.gpus_num)
+        self.sy_ = tf.split(probs, self.gpus_num)
+        self.sz_ = tf.split(winner, self.gpus_num)
         self.batch_norm_count = 0
-        self.y_conv, self.z_conv = self.construct_net(self.x)
+        self.reuse_var = None
 
-        if self.swa_enabled == True:
+        # You need to change the learning rate here if you are training
+        # from a self-play training set, for example start with 0.005 instead.
+        opt_op = tf.train.MomentumOptimizer(
```

Comment:
nit: this isn't an operation, so it probably shouldn't be called 'opt_op'. leaving it as the original 'opt' name would be better.